source/i18n.ts

```diff
@@ -166,8 +166,8 @@ function compileTemplates(root: Readonly<Record<string, string>>): RepositoryEnt
 
 /* eslint-disable @typescript-eslint/prefer-readonly-parameter-types */
 
-export function match(resourceKey: string, templateData: Readonly<TemplateData>): (text: string, ctx: TelegrafContextWithI18n) => string[] | null {
-  return (text, ctx) => (text && ctx?.i18n && text === ctx.i18n.t(resourceKey, templateData)) ? [text] : null
+export function match(resourceKey: string, templateData?: Readonly<TemplateData>): (text: string, ctx: TelegrafContextWithI18n) => string[] | null {
+  return (text, ctx) => (text && ctx?.i18n && text === ctx.i18n.t(resourceKey, templateData || {})) ? [text] : null
```

Comment:
the `ctx.i18n.t()` method already has an optional second argument so I would prefer using its way of handling the not provided templateData. So the match keeps to be a shortcut and `ctx.i18n.t()` knows how to deal with `undefined` as second argument.

https://github.com/EdJoPaTo/telegraf-i18n/blob/d1e03688f6ad76ae4b4a0cc58308a29c62304c1f/source/context.ts#L40

```suggestion
  return (text, ctx) => (text && ctx?.i18n && text === ctx.i18n.t(resourceKey, templateData)) ? [text] : null
```

Reply:
Yeah, ofc. I did only quick look over the code, so you can modify my commits as you wish.